### PR TITLE
Feature/emotion

### PR DESCRIPTION
**Pull request by phuongy**
I've started to convert some styles over to emotion, but its a big task. 

Let's agree on how we want to do this going forward, put any feedback / comments below.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,19 +6,24 @@
...
     "d.js": "^0.7.5",
+    "emotion": "^9.2.10",
+    "eslint-plugin-react": "^7.11.1",
     "gatsby": "2.0.7",
     "gatsby-plugin-react-helmet": "3.0.0",
     "makerule": "^1.0.5",
     "prop-types": "^15.6.2",
     "react": "^16.5.2",
     "react-dom": "^16.5.2",
+    "react-emotion": "^9.2.10",
```
Comment (antonjb) [inline package.json]: Is there value sticking within the Gatsby ecosystem?
https://www.gatsbyjs.org/packages/gatsby-plugin-emotion/

Comment (antonjb) [reply package.json]: e.g do we lose any Gatsby functionality by Emotion being outside the build tools?

File: src/components/EventCard.js [hunk 2]
```diff
@@ -30,3 +36,94 @@ EventCard.propTypes = {
...
+        if (p.isActive) return '#fff'
+
+        return `${COLOURS.PURPLE_LIGHT}`
+    }};
+    min-height: 160px;
+	transition: background 0.4s ease-in-out;
+
+    
+    &:hover,
+    &:focus {
```
Comment (antonjb) [inline src/components/EventCard.js]: I notice here :hover and :focus have & before them but they aren't there in src/components/Button.js, does that matter?

File: src/components/Events.js [hunk 3]
```diff
@@ -1,6 +1,22 @@
 import React from 'react'
+import styled from 'react-emotion'
 import { StaticQuery, graphql } from 'gatsby'
-import EventCard from './EventCard'
+import EventCard, { EventCardLink } from './EventCard'
+import { BREAKPOINTS } from '../constants'
+
+const determineType = name => {
```
Comment (antonjb) [inline src/components/Events.js]: Good idea moving this

**Comment by antonjb** [comment]
Nice work! It's good to see all the changes. I know you asked about approach, my preference would be that styles live in their own file that's imported but I'm not super opinionated on that, just how I have done it in the past.

File: package.json (hunk 1 above)
Comment (phuongy) [reply package.json]: Ok I will take a look, definitely didn't know about this.

**Comment by phuongy** [comment]
@antonjb Lets merge in the current changes and discuss the style / file structure thing in issue

**Pull request merged**